src/igl/opengl/RenderCommandEncoder.cpp

```diff
@@ -404,7 +404,6 @@ void RenderCommandEncoder::drawIndexed(size_t indexCount,
   (void)vertexOffset;
   (void)baseInstance;
 
-  IGL_ASSERT_MSG(instanceCount == 1, "Instancing is not implemented");
```

Comment:
Please split changes to this function into a separate small PR. It will be easier to review and merge everything that way.